.github/workflows/publish-snapshot.yml

```diff
@@ -41,7 +41,7 @@ jobs:
                       ${{ runner.os }}-
 
             - name: Install Dependencies
-              run: npm install
+              run: make setup
```

Comment:
no, the `npm install` was OK, you are just missing the `npx playwright install` here right after